tensorflow_probability/python/sts/decomposition.py

```diff
@@ -221,7 +221,7 @@ def decompose_by_component(model, observed_time_series, parameter_samples):
         model, posterior_means, posterior_covs, parameter_samples)
 
 
-def decompose_forecast_by_component(model, forecast_dist, parameter_samples):
+def decompose_forecast_by_component(model, forecast_dist, parameter_samples, observed_time_series):
```

Comment:
Adding a new argument would be a breaking change, and I'm not sure we actually need to do it. Would it work to pass `initial_step=forecast_lgssm.initial_step` below?

Reply:
Thank you for you review. I agree with it and changed the code as suggested.